src/main/resources/assets/api-docs.yaml

```diff
@@ -1738,6 +1729,28 @@ paths:
           description: DAR or User Not Found
         500:
           description: Internal Server Error.
+  /api/dar/draft:
```

Comment:
I'm guessing this is a typo/error? The draft endpoint exists on the V2Resource `/api/dar/v2/draft`, which already has documentation on this yaml file.